src/cdogs/actors.c

```diff
@@ -100,6 +100,7 @@
 #define GRIMACE_MELEE_TICKS 19
 #define DAMAGE_TEXT_DISTANCE_RESET_THRESHOLD (ACTOR_W / 2)
 #define FOOTPRINT_MAX 8
+#define GAME_TICKS_PER_SECOND 60 // Assuming 60 ticks per second for accumulatedDamage reset
```

Comment:
the game is 70FPS
please use FPS_FRAMELIMIT

Reply:
Fixed as advised